packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -75,28 +88,48 @@ const Summary: React.FunctionComponent< Props > = ( { siteId } ) => {
 			<div className="focused-launch-summary__step">
 				<div className="focused-launch-summary__data-input">
 					<div className="focused-launch-summary__section">
-						<DomainPicker
-							header={
-								<>
-									<label className="focused-launch-summary__label">
-										{ __( '2. Confirm your domain', __i18n_text_domain__ ) }
-									</label>
-									<p className="focused-launch-summary__mobile-commentary focused-launch-summary__mobile-only">
-										<Icon icon={ bulb } /> 46.9% of globally registered domains are .com
-									</p>
-								</>
-							}
-							existingSubdomain={ siteDomainName }
-							currentDomain={ siteDomainName }
-							onDomainSelect={ noop }
-							initialDomainSearch={ domainSearch }
-							showSearchField={ false }
-							analyticsFlowId="focused-launch"
-							analyticsUiAlgo="focused_launch_domain_picker"
-							onDomainSearchBlur={ () => noop( 'TODO: on domain search blur' ) }
-							onSetDomainSearch={ () => noop( 'TODO: on set domain search' ) }
-							quantity={ 3 }
-						/>
+						{ hasPaidDomain ? (
+							<>
+								<label className="focused-launch-summary__label">
+									{ __( 'Your domain', __i18n_text_domain__ ) }
+									<Tooltip
+										position="top center"
+										text={ __(
+											'Changes to your purchased domain can be managed from your Domains page.',
+											__i18n_text_domain__
+										) }
+									>
+										{ info }
+									</Tooltip>
```

Comment:
There is already a reusable component for this. It needs to be extracted to `@automattic/components`.
https://github.com/Automattic/wp-calypso/issues/45449